app/src/main/java/de/othr/plantico/ui/PlantViewModel.kt

```diff
@@ -1,19 +1,24 @@
 package de.othr.plantico
 
 import androidx.lifecycle.*
-import de.othr.plantico.database.PlanticoDB
 import de.othr.plantico.database.Repository
+import de.othr.plantico.database.entities.OwnedPlant
 import de.othr.plantico.database.entities.Plant
 import kotlinx.coroutines.launch
 
 class TestViewModel(private val repository: Repository) : ViewModel() {
```

Comment:
If the File is called PlantViewModel maybe we should also call the ViewModel PlantViewModel.